templates/core/stories.html

```diff
@@ -240,6 +264,10 @@ <h4>Cinergy makes significant digital signage saving with Screenly Pro and Ubunt
   </div>
 </section>
 
+<div class="u-fixed-width">
+  <hr class="p-separator" />
+</div>
```

Comment:
The spacing would look better here if this was within the same strip. I would wrap all of this together like the separator below